### ISSUE TITLE
Missing Tiny schematic in the given link

### ISSUE THREAD

**Issue by X-Ryl669** [RfidResearchGroup/ChameleonMini#67]
You are promoting the Chameleon Tiny and Mini as open source and provide schematic for the Mini. There is no schematic for the Tiny nor any PCB/Gerber. Do you have those and can you post them ?

Thanks!

**Comment by 0ff** [comment]
Yes, please post it so that we can see the connections between the ATMega and BLE Chip.
